### ISSUE TITLE
add Expense Tracker project

### ISSUE THREAD

**Issue by nagasaisriya** [Vishal-raj-1/Awesome-JavaScript-Projects#320]
![2021-03-11 (2)](https://user-images.githubusercontent.com/63009472/110680926-3aea2880-81ff-11eb-88d5-bf80ed37f809.png)

To Keep track of income and expenses, add and remove items and save to local storage.
I would like to contribute to this project by adding expense tracker as a part of gssoc'21.

**Comment by urvashi-code1255** [comment]
@nagasaisriya Expense Tracker issue is already made!!

Please get assigned to an issue before starting working on it .

**Comment by nagasaisriya** [comment]
I'm sorry! I thought we can directly make a pull request, wouldnt repeat it again.

**Comment by Vishal-raj-1** [comment]
@nagasaisriya Could You improve already made project ? If yes then do that, I will assign this issue to you

**Comment by nagasaisriya** [comment]
> @nagasaisriya Could You improve already made project ? If yes then do that, I will assign this issue to you

I would definitely try it @Vishal-raj-1 , can I get the issue link?

**Comment by Vishal-raj-1** [comment]
@nagasaisriya I mean You can check the project on the website and if you want to improve that project. UI or any other feature then you can do that and edit this issue title then I will assign the issue to you !! Otherwise you can check other issues as well

**Comment by urvashi-code1255** [comment]
@nagasaisriya Any updates on this?

**Comment by nagasaisriya** [comment]
Hey @urvashi-code1255 , Actually I did'nt understand what needs to be improved more, Do I need to improve the UI of the project commited by me?

**Comment by harshgupta20** [comment]
@nagasaisriya since this project already made so if we asked you to improve the UI or you can work on another issue if you want. I am closing it because this issue has been resolved already.
